tinker/__init__.py

```diff
@@ -114,42 +117,34 @@ def get_url_from_path(path, **kwargs):
 
 
 # New importing of routes and blueprints
-from tinker.views import BaseBlueprint
-from tinker.admin.cache import CacheBlueprint
-from tinker.admin.blink_roles import BlinkRolesBlueprint
-from tinker.admin.program_search import ProgramSearchBlueprint
-from tinker.admin.sync import SyncBlueprint
-from tinker.admin.publish import PublishBlueprint
-from tinker.admin.program_search import ProgramSearchBlueprint
-from tinker.admin.redirects import RedirectsBlueprint
-from tinker.e_announcements import EAnnouncementsBlueprint
-from tinker.faculty_bios import FacultyBiosBlueprint
-from tinker.office_hours import OfficeHoursBlueprint
-from tinker.events import EventsBlueprint
-from tinker.news import NewsBlueprint
-from tinker.admin.user_roles import UserRolesBlueprint
-
-app.register_blueprint(BaseBlueprint)
-app.register_blueprint(CacheBlueprint)
-app.register_blueprint(BlinkRolesBlueprint)
-app.register_blueprint(ProgramSearchBlueprint)
-app.register_blueprint(SyncBlueprint)
-app.register_blueprint(PublishBlueprint)
-app.register_blueprint(ProgramSearchBlueprint)
-app.register_blueprint(RedirectsBlueprint)
-app.register_blueprint(EAnnouncementsBlueprint)
-app.register_blueprint(FacultyBiosBlueprint)
-app.register_blueprint(OfficeHoursBlueprint)
-app.register_blueprint(EventsBlueprint)
-app.register_blueprint(NewsBlueprint)
-app.register_blueprint(UserRolesBlueprint)
-
-
-from tinker.unit_test_interface import UnitTestBlueprint
```

Comment:
Left side, lines 148 and 149: what happened to the unit_test_interface import?

Reply:
Fixed.